src/Microsoft.AspNetCore.BuildTools.ApiCheck/ApiListingComparer.cs

```diff
@@ -13,128 +13,108 @@ public class ApiListingComparer
     {
         private readonly ApiListing _newApiListing;
         private readonly ApiListing _oldApiListing;
-        private readonly IList<ApiChangeExclusion> _exclusions;
 
         public ApiListingComparer(
             ApiListing oldApiListing,
-            ApiListing newApiListing,
-            IList<ApiChangeExclusion> exclusions = null)
+            ApiListing newApiListing)
         {
             _oldApiListing = oldApiListing;
             _newApiListing = newApiListing;
-            _exclusions = exclusions ?? new List<ApiChangeExclusion>();
         }
 
-        public ApiComparisonResult GetDifferences()
+        public IList<BreakingChange> GetDifferences()
```

Comment:
@Eilon I asked @javiercn about the errors output for extraneous exceptions. He said you had wanted that feature and considered the extra exceptions errors. IIRC the purpose was to make sure we'd reported all _real_ breaking changes at the end of a milestone. Has something changed?